src/aap_eda/wsapi/consumers.py

```diff
@@ -148,19 +147,14 @@ def handle_heartbeat(self, message: HeartbeatMessage) -> None:
         ).first()
 
         if instance:
-            instance.status = ActivationStatus.RUNNING
             instance.updated_at = message.reported_at
-            instance.save(update_fields=["status", "updated_at"])
+            instance.save(update_fields=["updated_at"])
```

Comment:
@bzwei we should do this in a separate PR

Reply:
This change has been extracted in #435.